adhocracy-plus/assets/scss/components/_header_lower.scss

```diff
@@ -21,6 +21,13 @@
     box-shadow: 0 0 8px 0 $shadow;
 }
 
+.header-lower__brand {
```

Comment:
same here: should the header changes also be in a+?

Reply:
same as above